internal/daemon/ucrednet.go

```diff
@@ -27,7 +27,9 @@ import (
 var errNoID = errors.New("no pid/uid found")
 
 const (
-	ucrednetNoProcess = int32(0)
+	// ucrednetNoProcess is -1 to avoid the case when pid=0 is returned from the kernel
+	// because the connecting process is in an unrelated process namespace.
+	ucrednetNoProcess = int32(-1)
```

Comment:
Is this safe to use or are PIDs in a positive 32bit namespace? /me checks
```
$ cat /proc/sys/kernel/pid_max
4194304
```
Alright, this should be okay.